llpc/test/shaderdb/gfx10/CheckFMFOptions_NoContract.pipe

```diff
@@ -0,0 +1,220 @@
+// This test checks that setting disableFastMathFlags to 32 (1<<5 == AllowContract)
+// does actually stop contract (no fma instructions are formed)
+// See CheckFMFOptions_default.pipe for the default case
```

Comment:
ubernit: could you finish sentences with `.`?

Reply:
Done